docs/docs/scopes.md

```diff
@@ -106,6 +106,7 @@ Project.scope(['deleted', 'activeUsers']).findAll();
 ```sql
 SELECT * FROM projects
 INNER JOIN users ON projects.userId = users.id
+WHERE deleted = true
```

Comment:
Before this condition was on the JOIN, now it is inside the WHERE. Can you confirm that this is actually the case?